### DISCUSSION TITLE
Spell Effect [Serious Injury]

### DISCUSSION THREAD

**Discussion by Kamusvalenzuela** [AscensionGameDev/Intersect-Engine#1488] [Ideas]
**Is your feature request related to a problem? Please describe.**
A Spell effect to reduce healing received in porcentages or something similar. can be a really nice thing to add to the engine

**Comment by Spectreye** [comment]
Adding the opposite effect by the way would be nice too 
ex: +25% heal received.

**Comment by Kamusvalenzuela** [comment]
Yeah!
the both are super effective in any RPG
+X% heal /// -X% heal

**Comment by WeylonSantana** [comment]
I've been fiddling with the effects a lot these days, I believe I could fall for this one, what would the spell effect name for this one?

**Reply by PyroTech03** [reply]
Wounded for beg and revitalized for positive?

**Reply by Ezekyuhl** [reply]
This could be useful for att/def too, i think in 14 it is called "Vulnerability"
